packages/block-editor/src/components/inspector-controls/slot.js

```diff
@@ -7,8 +7,11 @@ import warning from '@wordpress/warning';
 /**
  * Internal dependencies
  */
+import BlockSupportToolsPanel from './block-support-tools-panel';
 import groups from './groups';
 
+const blockSupportGroups = [ 'border', 'color', 'dimensions', 'typography' ];
```

Comment:
Is it worth us using a shared constant between these strings and the ones in `block-support-tools-panel` to ensure they match?

Reply:
That's looking good now, thanks for tightening this up! 👍

Reply:
It definitely could be. 

For now, I've opted to only tweak this such that the labels object is used for this check. I think the benefit of the shared constants would come in if they are adopted when [defining the groups](https://github.com/WordPress/gutenberg/blob/7c23fd8d87361eee366797998cc36a9c2f40e17a/packages/block-editor/src/components/inspector-controls/groups.js) and consuming the API:
e.g. `<InspectorControls group={ GROUP_DIMENSIONS }>`

I'm happy to add the constants in a follow up PR to this and https://github.com/WordPress/gutenberg/pull/34069

Reply:
Sounds good to me, the way you've combined them addresses the initial concern I had, and agreed, the individual / shared constants will be more useful when defining the groups and consuming the API as you suggest 👍. I'm happy to leave any other changes to follow-ups!